python/Dockerfile

```diff
@@ -6,6 +6,9 @@ RUN wget https://code-server.dev/install.sh -O /tmp/install.sh && \
     sh /tmp/install.sh --version 4.19.1 && \
     rm /tmp/install.sh
 
+RUN LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH \
+    pip install git+https://github.com/opendatacube/odc-algo.git@add-rust-geomedian-impl
```

Comment:
Do you need to `conda activate <env>` first before running `pip install`?

Reply:
Or actually, you could put `odc-algo` in the `environment.yml` file right?

Reply:
Yeah, it should be in requirements. I was testing to see if installing as root makes a difference.